test/freenet/support/PrioritizedTickerTest.java

```diff
@@ -55,6 +67,71 @@ public void testSimple() throws InterruptedException {
 		}
 	}
 
+    public void testRemove() throws InterruptedException {
```

Comment:
Unfortunately most of these issues are hard to fix without changing the code being tested. I will look at it shortly, I don't think this test is doing what it's supposed to do at the moment - that is, I don't see why it is throwing an assertion error in these cases unless there is a leak...